netcore/NetCore.Test/CacheFactoryUnitTest.cs

```diff
@@ -0,0 +1,81 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+using System;
+using Apache.Geode.Client;
+using Xunit;
+
+namespace GeodeDotNetTest {
+  [Collection("Geode .net Core Collection")]
+  public class CacheFactoryUnitTests {
+    [Fact]
+    public void TestCreateFactory() {
```

Comment:
Since all these methods are tests, does it make sense to redundantly prefix them all with `Test`? If read in plain English you have "Fact, test create factory." This is is not a complete fact. More appropriately it should be called `CacheFactoryCreateIsNotNull` read as "Fact, CacheFactory.Create is not null", which is an asserted fact of this test.

Reply:
Removed `Test` prefix, and `Unit` from test class and test filename.  Added some more asserts, and made test case names a bit more descriptive where warranted.